toolkit/components/uniffi-bindgen-gecko-js/src/templates/Scaffolding.cpp

```diff
@@ -208,4 +210,22 @@ RootedDictionary<UniFFIRustCallResult>& aUniFFIReturnValue, ErrorResult& aUniFFI
 
 {%- endfor %}
 
+
+{%- for object in ci.iter_object_definitions() %}
+already_AddRefed<UniFFIPointer> {{ ci.scaffolding_class() }}::ReadPointer{{ object.nm() }}(const GlobalObject& aUniffiGlobal,
+ const ArrayBuffer& buff,
+ const int64_t& position) {
+     //TODO: Figure out the "right" way to read an array buffer in C++
```

Comment:
just want to flag this so I remember it when testing the "pointer from buffer" flow, it probably doesn't even work properly now